docs/BEST_PRACTICES.md

```diff
@@ -1,3 +1,69 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Order creation best practices](#order-creation-best-practices)
+    - [AFTER a successful payment](#after-a-successful-payment)
+      - [Limitations & Solutions](#limitations--solutions)
+    - [BEFORE a successful payment](#before-a-successful-payment)
+      - [Limitations](#limitations)
+      - [Conclusion](#conclusion)
+- [Payment related best practices](#payment-related-best-practices)
+- [Order/Cart related best practices](#ordercart-related-best-practices)
+- [Deployment best practices](#deployment-best-practices)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Order creation best practices
+#### AFTER a successful payment
+##### Limitations & Solutions
+- It is possible that shop was not reachable for network issues. In that case there will be a successful payment but no order.
+
+  ###### Possible solution:
+  Create an order based on transaction state changes (commercetools messages or subscriptions)
+
+------  
+- It is possible to have more than 1 valid payments on cart/order. During checkout in two tabs, two redirect payments
+(like credit card and paypal) can be created for the same cart. These two payments can be completed independently in both tabs.
+ This will lead the cart that links to two successful payments.
+  ###### Possible solution:
+  Refund the payment. (It can be based on commercetools messages or subscriptions)
+
+------  
+- It is possible to have the payment amount is lower than cart amount. During the checkout the shopper opens two tabs, one tab is for cart and
```

Comment:
In general I would try to separate everywhere the issue title which is `It is possible to have the payment amount is lower than cart amount` from the detailed description of it and maybe make it possible to link so that one could share a deep link as also link directly within our documentation. Also I would rephrase the title to: `The amount of the successful payment is lower than the cart amount`

Reply:
d8f39a0726026abab680a9dd9a3fec009a21e10f

Reply:
Thanks. I had another look at the end result here: https://github.com/commercetools/commercetools-adyen-integration/blob/d8f39a0726026abab680a9dd9a3fec009a21e10f/docs/BEST_PRACTICES.md#more-than-1-valid-payments-on-the-cartorder

maybe it would be better:
- Not to use a bullet point where there is a single point only
- Do not use grey line separator between the bullet points here: https://github.com/commercetools/commercetools-adyen-integration/blob/d8f39a0726026abab680a9dd9a3fec009a21e10f/docs/BEST_PRACTICES.md#before-a-successful-payment

Reply:
a74dbc4